cas-server-support-saml/src/test/resources/META-INF/spring/opensaml-config.xml

```diff
@@ -32,7 +32,7 @@
                            http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd
                            http://www.springframework.org/schema/mvc http://www.springframework.org/schema/mvc/spring-mvc.xsd">
 
-    <context:component-scan base-package="org.jasig.cas.support.saml" annotation-config="true" />
+    <context:component-scan base-package="org.jasig.cas" annotation-config="true" />
```

Comment:
Needs to be removed, since we already have an annotation-config element. Will do.